lib/board.rb

```diff
@@ -1,5 +1,7 @@
 class Board
   attr_reader :board
+  attr_reader :POSITION_MIN
```

Comment:
I used to avoid my variable on the same line, I don't know if it's a good or bad habits